src/App.css

```
@@ -0,0 +1,180 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Pretendard'
+}
+
+header{
+  display: flex;
+  width: 100%;
+  align-items: center; 
+}
+
+header img{
+  margin-left : 200px;
+}
+
+nav { 
+  display:flex;
+  align-items: center;
+  margin-left: 32px;
+}
+
+nav a{
+  font-size:18px;
+  font-weight: 700;
+  text-decoration: none;
+  color: black;
+  width:120px;
+  height:68px;
+  padding:21px 15px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center; 
+}
+main{
+  margin-left:360px;
+  margin-right:360px;
+  margin-top:94px;
+  margin-bottom: 141px;
+}
+
+main h1{
+  font-size:20px;
+  line-height: 32px;
+  margin-bottom: 12px;
+}
+
+
+.profile{
+  position:absolute;
+  right:200px;
+}
+.best{
+  display:flex;
+  width: 100%;
+}
+.best div{
+  margin-right:24px;
+}
+.itemTitle{
+  font-size: 14px;
+  line-height: 24px;
+  font-weight: 500;
+}
+.price {
+  font-weight: 700;
+  font-size:16px;
+  line-height: 26px;
+}
+.favorite_icon{
+  display:flex;
+  align-items: center;
+}
+.favorite_icon img{
+  width: 13.4px;
+  height:11.65px;
+  margin-right:3px;
+}
+
+.entireHeader{
+  display:flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 40px;
+}
+
+.entireHeader div{
+  display:flex;
+}
+
+.searchInput{
+  width: 325px;
+  height:42px;
+  border-radius: 12px;
+  background-color:#F3F4F6;
+  background-image: url('/src/assets/Vector.png');
+  border : 1px solid #ffffff;
+  background-repeat: no-repeat;
+  background-position: 10px center;
+  margin-right:12px;
+  padding-left:30px;
+}
+.RegistButton{
+  width: 133px;
+  height:42px;
+  border-radius: 8px;
+  background-color: #3692FF;
+  border: 1px solid #ffffff;
+  color : #ffffff;
+
+  font-weight: 600;
+  font-size:16px;
+  line-height: 26px;
+  margin-right:12px;
+  padding: 8px 14px;
+  text-decoration: none;
+}
+
+.Recently_favorite_sortmenu{
+  background-color: #FFFFFF;
+  border: 1px solid #E5E7E8;
+  width: 130px;
+  height: 42px;
+  border-radius: 12px;
+  display: flex; 
+  align-items: center; 
+  justify-content: space-between; 
+  padding: 12px 20px; 
+}
+
+.dropdown_menu{
+  position:absolute;
+  display:flex;
+  flex-direction: column;
+  align-items: flex-end;
+  margin-top:42px;
+  right: 360px;
```

P2:
드롭다운 메뉴 위치가 좀 다르게 작업된 것 같습니다.
위치값을 아래와 같이 바꾸고
`.Recently_favorite_sortmenu` 컴포넌트에 position: relative 추가해보세요.

```suggestion
  right: 0;
  top: 0;
```